src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const INVERSE_KEYS_AND_VALUES = {};
+
+  for (const key in robot) {
+    if (INVERSE_KEYS_AND_VALUES[robot[key]] !== undefined) {
```

Comment:
How about this object, `const robot = {"a": 123, "b": undefined};`, it will fail your function. Read the second point of [checklist](https://github.com/mate-academy/js_inverse_robot/blob/master/checklist.md)